db/seeds.rb

```diff
@@ -1,4 +1,4 @@
-raise if Rails.env.production?
+raise "Aborting seeds - running in production with existing vacancies" if Rails.env.production? && Vacancy.any?
```

Comment:
or maybe check that `DOMAIN != PROD_DOMAIN` or something like that? I am ok with this too

Reply:
I was thinking about that, or adding an `APP_ROLE` env var or something – but this strikes me as a good balance because it'll cover us without any extra effort even if we change domains etc 🤞